airbyte-webapp/src/pages/DestinationPage/DestinationPage.tsx

```diff
@@ -18,6 +19,10 @@ const Content = styled.div`
 `;
 
 const DestinationPage: React.FC = () => {
+  useEffect(() => {
+    AnalyticsService.page("Destination Page");
```

Comment:
Is it expected that you are logging page here but also in the routing?

Ideally the page call should only happen in routing so there is a central place for logging pages.

Reply:
This one was added by accident.

Ideally, we should track it in 1 place, but we have it in 2 places: 1 for users with connection ( passed onboarding ) and 1 without. Such behaviour was in the previous project and as I remember that was because of needs to distinguish logged in and non-logged in user, so I've gone with the same way here too.